share/qt/clean_mac_info_plist.py

```diff
@@ -5,25 +5,33 @@
 
 from string import Template
 from datetime import date
+import os
 
-bitcoinDir = "./";
+bitcoin_directory =  os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+version = "unknown"
 
-inFile     = bitcoinDir+"/share/qt/Info.plist"
-outFile    = "Bitcoin-Qt.app/Contents/Info.plist"
-version    = "unknown";
 
-fileForGrabbingVersion = bitcoinDir+"bitcoin-qt.pro"
-for line in open(fileForGrabbingVersion):
-	lineArr = line.replace(" ", "").split("=");
-	if lineArr[0].startswith("VERSION"):
-		version = lineArr[1].replace("\n", "");
+''' If the output folder doesn't exists? create it.'''
+output_directory = "Bitcoin-Qt.app/Contents"
```

Comment:
`output_directory = dimecoin_directory + "Bitcoin-Qt.app/Contents"`

Reply:
we should probably change `Bitcoin-Qt.app` to `Dimecoin-Qt.app` here and in `fancy.plist`